cms/templates/library.html

```diff
@@ -27,7 +27,9 @@
             {
                 isUnitPage: false,
                 page_size: 10,
-                canEdit: ${"true" if can_edit else "false"}
+                canEdit: ${"true" if can_edit else "false"},
+                togglePreviews: true,
```

Comment:
Is this togglePreviews variable ever used? I only see togglePreviews as a function.